### PR TITLE
Script templates (ruby, python, node, bash)

### PR DESCRIPTION
**Pull request by mhgoldman**

### CODE REVIEW

File: script-templates/python/lib/script_support/lab_control.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+import json
+from skytap_metadata import SkytapMetadata
+
+class LabControl:
```
Comment (maxwellbooth) [inline script-templates/python/lib/script_support/lab_control.py:19]: I haven't used python so this syntax all looks a bit strange to me, but it seems like it should work...

**Review by maxwellbooth — APPROVED** [review]
Looks good! I made a few comments for minor things but I think it seems fine overall.

File: script-templates/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+From there, try one of the following:
+
+```
+curl -s -X POST $BROADCAST_URL -d '{"type":"refresh_content_pane"}'
+curl -s -X POST $BROADCAST_URL -d '{"type":"refresh_lab"}'
+```
+
+### Metadata Stub Service
+
+A chellenge in developing scripts that interact with lab metadata is that it is only available from within a Skytap environment. To help with this, the `bin/run` script runs a "metadata stub" service, simulating the behavior of the Metadata Service and Control Endpoint locally and returning stubbed data. If you would like to modify the stubbed data returned when running your script locally, copy the `../lib/metadata_stub_server/lib/stub_data` directory to `lib/script_support/stub_data` and modify the files accordingly.
```
[the remark this answers is not in the document]
Comment (mhgoldman) [reply script-templates/bash/README.md:88]: Thanks, will fix.

File: script-templates/bash/lib/template_support/build [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (mhgoldman) [reply script-templates/bash/lib/template_support/build:1]: Yes, it's intended to be a template so you could add logic here if you needed to.

File: script-templates/python/lib/script_support/lab_control.py (hunk 1 above)
Comment (mhgoldman) [reply script-templates/python/lib/script_support/lab_control.py:19]: This was the first python I've ever written, and I'm also not a fan, but it seems to work!

**Pull request merged**